hack/v1migration/validate.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+WORKDIR=${1:?must specify working directory}
+echo "Using workdir: $WORKDIR"
+
+if [[ ! -d "$WORKDIR" ]]
+then
+  echo "Directory $WORKDIR does not exist."
+  exit 1
+fi
+
+# shellcheck source=hivetypes.sh
+source "$(dirname "$0")/hivetypes.sh"
+
+TEMPDIR="$(mktemp -d)"
+
+for t in "${HIVE_TYPES[@]}"
+do
+  if [[ "$t" == "dnsendpoints" ]]; then continue; fi
+
+  echo "Validating ${t}"
+
+  <"${WORKDIR}/$t.json" jq '.metadata.namespace + "/" + .metadata.name' | sort > "${TEMPDIR}/$t.orig"
```

Comment:
Same question as above about redirecting instead of using filename at the end of the command.